docs/UserGuide.md

```diff
@@ -366,7 +385,7 @@ Examples:
 `change da/2021-12-28`  
 
 
-#### Viewing schedule of staff(s): `viewSchedule`
+#### Viewing schedule of staff: `viewSchedule`
```

Comment:
I don't really understand the problem with this? viewSchedule command can view the schedule of multiple staff?

Reply:
It was just a UG bug someone reported, but I can revert if if ya want?

Reply:
I think the typo in #167 is referring to `view the schedule of staff's ...` should be `view the schedule of staffs ...`

Reply:
Ohh yea that makes sense, thank you!